bedrock/settings/base.py

```diff
@@ -110,9 +111,11 @@ def path(*args):
 
 # Tells the product_details module where to find our local JSON files.
 # This ultimately controls how LANGUAGES are constructed.
-PROD_DETAILS_DIR = path('lib', 'product_details_json')
 PROD_DETAILS_CACHE_NAME = 'product-details'
 PROD_DETAILS_CACHE_TIMEOUT = 60 * 15  # 15 min
+default_pdstorage = 'PDDatabaseStorage' if PROD else 'PDFileStorage'
```

Comment:
I'm not sure if "PROD" is the right name for the setting, since I would want to default to the db for dev and demo instances as well, but in the interests of time I'm willing to merge this as-is until we can come up with a better name.